expo/expo
# fix: package names involving kotlin keywords

# Why

Package names could not use specific strings until now. That includes the Islandic TLD `.is`, which for a package name goes to `is.pvin.appname`, and `is` is a kotlin keyword, so we need to escape it.

More info: https://github.com/expo/expo/issues/26910.

Solves https://github.com/expo/expo/issues/26910.

# How

When we are about to replace the package name on a kotlin file, we make sure to escape the right strings. We can add more in that array.

# Test Plan

- Create a new expo project
- Add a package name in `app.json` like:
```
"android": {
  "package": "is.wow.fun.lol"
}
```
- Run expo prebuild -p android to generate a new android project (remove existing one if you have an android directory)
- Make sure `MainActivity.kt` and `MainApplication.kt` have the package name at the top of the file with wrapped `is` and `fun`

# Checklist
